components/automate-deployment/pkg/preflight/checks.go

```diff
@@ -403,7 +403,11 @@ func requiredPortsForConfig(skipShared bool, config *dc.AutomateConfig) ([]int,
 	portsToCheck = append(portsToCheck, 9631, 9638)
 	acValue := reflect.ValueOf(*config)
 	for i := 0; i < acValue.NumField(); i++ {
-		if v, ok := acValue.Field(i).Interface().(a2conf.A2ServiceConfig); ok {
+		f := acValue.Field(i)
+		if !f.CanInterface() {
```

Comment:
this skips private struct fields. with the older proto code there weren't any, but now there are.